torch/onnx/_internal/fx/passes/modularization.py

```diff
@@ -818,6 +835,17 @@ class Modularize(_pass.Transform):
 
     """
 
+    @_beartype.beartype
```

Comment:
The docstring above should mention that modularization works for both torch.export.ExportedProgram and torch.nn.Module